buffer/src/test/java/io/netty/buffer/AdaptiveByteBufAllocatorTest.java

```diff
@@ -153,6 +153,9 @@ public void run() {
             }).start();
         }
         countDownLatch.await();
-        assertNull(throwableAtomicReference.get());
+        Throwable throwable = throwableAtomicReference.get();
```

Comment:
nit we could also just throw it directly

Reply:
I disagree, as it would loose the stack trace of where in the test we _propagated_ it. 🙂